pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
"loo[p]# sounds like a great way to incorporate an explanation for `-p`! @agnivade what do you think of `Loo[p] over all lines of a file, editing them [i]n-place using a find/replace [e]xpression:`?